client/main.lua

```diff
@@ -66,8 +66,11 @@ AddTextEntry('FE_THDR_GTAO', mapText)
 
 CreateThread(function()
     for _, v in pairs(GetVehiclesByName()) do
-        if v.hash and v.name then
-			AddTextEntryByHash(v.hash, v.name)
+        if v.model and v.name then
+            local gameName = GetDisplayNameFromVehicleModel(v.model)
+            if gameName or gameName ~= "CARNOTFOUND" then
```

Comment:
```suggestion
            if gameName and gameName ~= "CARNOTFOUND" then
```
I'm assuming this is supposed to be an and. It doesn't make sense otherwise

Reply:
woopsies, i was very tired 😅